src/emailservice/requirements.txt

```diff
@@ -7,18 +7,19 @@ configparser==3.5.0
 cryptography==2.3.1
 entrypoints==0.2.3
 enum34==1.1.6
-futures==3.2.0
+futures==3.1.1
 google-api-core==1.4.0
 google-auth==1.5.1
 google-cloud-core==0.28.1
 google-cloud-trace==0.19.0
 googleapis-common-protos==1.5.3
 grpc-google-iam-v1==0.11.4
-grpcio==1.15.0
-grpcio-health-checking==1.15.0
-grpcio-tools==1.15.0
+grpcio==1.12.1
+grpcio-health-checking==1.14.1
```

Comment:
ideally this also should be 1.12.1 but it's probably identical to 1.14.1.

Reply:
Agreed. The previous `requirements.txt` had it at `1.14.1` so I kept it at that. I'll downgrade it and test